src/Bootstrappers/Integrations/PostgresTenancyBootstrapper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Bootstrappers\Integrations;
+
+use Illuminate\Contracts\Config\Repository;
+use Illuminate\Database\DatabaseManager;
+use Stancl\Tenancy\Contracts\TenancyBootstrapper;
+use Stancl\Tenancy\Contracts\Tenant;
+use Stancl\Tenancy\Database\Contracts\TenantWithDatabase;
+
+class PostgresTenancyBootstrapper implements TenancyBootstrapper
```

Comment:
Is this used instead of `DatabaseTenancyBootstrapper` or alongside it?

Reply:
Instead, since we only use `DatabaseTenancyBootstrapper` with multi-db tenancy. I'll update the bootstrapper's docblock to make this clear

Reply:
Think we should rename the class, right now it sounds like something that people could enable when they're using postgres. To be clear: this is ONLY used when using RLS, right?

If so, maybe something like `PostgresRLSBootstrapper` could work better (or anything similar to that), so that it's explicitly in the name.

```
/**
 * Purge central connection, set Postgres credentials to the tenant's credentials
 * and use Postgres as the central connection.
 *
 * This bootstrapper is intended to be used with single-database tenancy.
 */
```

Is RLS "triggered" by using the user (that it's configured/enforced for) in the DB connection?

Reply:
> To be clear: this is ONLY used when using RLS, right?

When using the single-database approach – no matter if RLS is used

> Is RLS "triggered" by using the user (that it's configured/enforced for) in the DB connection?

RLS is enabled (or triggered, I think) for specific tables by creating the policies using CreateRLSPoliciesForTenantTables. The user in the DB connection is used for checking the RLS constraint (checking if the record's `tenant_id`/column name returned by `tenancy()->tenantKeyColumn()` matches the ID of the current user – maybe we could make the column name configurable, since we made the username more customizable using `$tenant->database()->getUsername() ?? $this->tenant->getTenantKey()`)

I thought about changing the bootstrapper's name too, but a name suggesting that RLS is the only thing this is used for doesn't seem right – the most correct name would be something like PostgresSingleDatabaseBootstrapper.

Reply:
For the record, I've renamed the bootstrapper to PostgresRLSBootstrapper, updated the annotation and simplified the bootstrapper a bit

https://github.com/archtechx/tenancy/pull/1105/commits/12b36f407d9ae557bb648885b03225a5c782e9fa